### PR TITLE
docs: improvements in the handling of transformations

### PR DESCRIPTION
**Pull request by mrelemerson**
The transformations are performed with the change-case and pluralize libraries, as well as allowing to combine them with each other. See the documentation. Also, I consider it a major update as it changes and/or removes some transformations (names, behavior).

### CODE REVIEW

**Comment by Huuums** [comment]
Moin @mrelemerson, thanks for the PR!

Overall I really like your changes. However if possible I would like to not introduce another breaking change so soon again. 

Would it be alright with you to handle the old transformer names and translate them into the new ones? 

e.g. `lowercase` should still work for the enduser but internally it will be translated to `lowerCase`  to work with the `change-case` library. 

That way we won't break the extension for every single user out there but can still take advantage of the `change-case` library.

**Comment by mrelemerson** [comment]
Hi @Huuums, yes I will change it to maintain compatibility.

**Comment by mrelemerson** [comment]
ready

File: src/lib/stringHelpers.ts [hunk 1]
```diff
@@ -33,31 +35,36 @@ const getTransformedSSFName = (replacement: string, transformer: string) =>
...
-  .caseEqual('pascalcase', pascalCase(replacement))
+  .caseEqual('pascalcase', pascalCase(replacement, { transform: pascalCaseTransformMerge }))
   .caseEqual('pathcase', pathCase(replacement))
   .caseEqual('sentencecase', sentenceCase(replacement))
   .caseEqual('snakecase', snakeCase(replacement))
   .caseEqual('singular', singular(replacement))
   .caseEqual('plural', plural(replacement))
   .caseEqual('lowercasefirstchar', lowerCaseFirstChar(replacement))
-  .caseEqual('capitalize', capitalCase(replacement))
+  .caseEqual('capitalize', capitalize(replacement))
```
Comment (Huuums) [inline src/lib/stringHelpers.ts:56]: capitalCase behaves differently from what capitalize did so this was reverted

Comment (mrelemerson) [reply src/lib/stringHelpers.ts:56]: ok

**Pull request merged**